cg/src/domains/processors/processors.service.js

```diff
@@ -0,0 +1,11 @@
+const { db } = require('../../db');
+
+class ProcessorsService {
+  constructor(database = db) {
+    this.db = database;
+  }
```

Comment:
This is empty for now, we may change it in the future if we decide to automate the entire flow of adding a processor to the network